src/cloudai/systems/slurm/slurm_system.py

```
@@ -341,9 +370,11 @@ def get_job_status(self, job: BaseJob, retry_threshold: int = 3) -> list[SlurmSt
             stdout, stderr = self.cmd_shell.execute(command).communicate()
             logging.debug(f"Job status: {command=} {stdout=} {stderr=}")
 
-            if "Socket timed out" in stderr or "slurm_load_jobs error" in stderr:
+            if self._is_transient_status_error(stderr):
                 retry_count += 1
                 logging.warning(f"Retrying job status check (attempt {retry_count}/{retry_threshold})")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)